doc/reference/tooling/tt_cli/connect.rst

```diff
@@ -38,14 +68,50 @@ You can also connect to instances in the same ``tt`` environment
 (that is, those that use the same :ref:`configuration file <tt-config_file>` and Tarantool installation)
 by their instance names.
 
-If authentication is required, specify the username and the password using the ``-u`` (``--username``)
-and ``-p`` (``--password``) options.
+Authentication
+~~~~~~~~~~~~~~
+
+If authentication is required, use one of the following ways to pass the username and
+the password:
+
+*   The ``-u`` (``--username``) and ``-p`` (``--password``) options:
 
-By default, ``tt connect`` opens an interactive Tarantool console. Alternatively, you
-can open a connection to evaluate a Lua script from a file or stdin. To do this,
-pass the file path in the ``-f`` (``--file``) option or use ``-f -`` to take the script
-from stdin.
+..  code-block:: console
+
+    $ tt connect 192.168.10.10:3301 -u myuser -p p4$$w0rD
+
+*   The connection string:
+
+..  code-block:: console
+
+    $ tt connect myuser:p4$$w0rD@192.168.10.10:3301 -u myuser -p p4$$w0rD
+
+*   Environment variables ``TT_CLI_USERNAME`` and ``TT_CLI_PASSWORD`` :
+
+..  code-block:: console
+
+    $ export TT_CLI_USERNAME=myuser
+    $ export TT_CLI_PASSWORD=p4$$w0rD
+    $ tt connect 192.168.10.10:3301
+
+Encrypted connection
+~~~~~~~~~~~~~~~~~~~~
+
+To connect to instances that use SSL encryption, provide the SSL certificate path and
+other encryption parameters in the options ``--sslcertfile``, ``--sslcafile``,
+``--sslkeyfile``, and ``--sslciphers``.
```

Comment:
Strictly speaking, `--sslcertfile` and `--sslkeyfile` are mandatory. The other parameters are optional.